Notebooks/Machine Learning with Swift for TensorFlow and SwiftPlot.ipynb

```diff
@@ -1167,6 +1189,8 @@
     "\n",
     "lossGraph.addSeries(epochs, trainLoss, label: \"Train\", color: .orange)\n",
     "lossGraph.addSeries(epochs, testLoss, label: \"Test\", color: .lightBlue)\n",
+    "let scaledTestLoss = testLoss.map {$0 * 6}\n",
```

Comment:
Rather than applying a map to perform a sixfold scaling here and introduce a third series, I'd highly recommend applying loss averaging at the point above where losses are added to the array.